sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/IfCollapsible.vb

```diff
@@ -0,0 +1,43 @@
+﻿Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+Imports System.Text
+
+Namespace Tests.TestCases
+    Class Foo
+        Public Sub Test(ByVal cond1 As Boolean, ByVal cond2 As Boolean, ByVal cond3 As Boolean)
```

Comment:
I have added a test. It would make more sense for it to raise an issue but it doesn't, which mimics the C# behavior so I'd rather keep it like this.

I am going to create an improvement ticket.